test/common/rhmi_config.go

```diff
@@ -0,0 +1,55 @@
+package common
+
+import (
+	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"testing"
+
+	goctx "context"
+)
+
+const (
+	RHMIConfigCRName  = "rhmi-config"
+	OperatorNamespace = "redhat-rhmi-operator"
+)
+
+// TestIntegreatlyRoutesExist tests that the routes for all the products are created
+func TestRHMIConfigCRs(t *testing.T, ctx *TestingContext) {
+	t.Log("Test rhmi config cr creation")
+
+	rhmiConfig := &v1alpha1.RHMIConfig{
+		TypeMeta: v1.TypeMeta{},
+		ObjectMeta: v1.ObjectMeta{
+			Name:      RHMIConfigCRName,
+			Namespace: OperatorNamespace,
+		},
+	}
+
+	if err := ctx.Client.Create(goctx.TODO(), rhmiConfig); err != nil {
+		if err != nil {
+			t.Fatalf("Failed to create RHMI Config resource %v", err)
+			return
+		}
+	}
```

Comment:
```suggestion
	if err := ctx.Client.Create(goctx.TODO(), rhmiConfig); err != nil {
		t.Fatalf("Failed to create RHMI Config resource %v", err)
	}
```
I'm not sure is the inner `if` and `return` neccessary here? :thinking:

Reply:
Also what do you think on deleting the resource after the test so the tests can be re-run? Would fail on an `already exists` error if re-run again :thinking:

Reply:
Yep, will do that. thanks